README.md

```diff
@@ -152,18 +165,18 @@ If you want to automatically format your code on every commit, you can use [pre-
 Just install it via `pip install pre-commit` and execute `pre-commit install` in the root folder.
 This will add a hook to the repository, which reformats files on every commit.
 
-If you want to set it up manually, install black via `pip install black`.
+If you want to set it up manually, install black via `pip install -r requirements-dev.txt`.
 To reformat files execute
 ```
-black .
+make formatter
 ```
 
 #### Type Checking
 
-If you want to check types on the codebase, install `pytype` using `pip install pytype`.
+If you want to check types on the codebase, install `pytype` using `pip install -r requirements-dev.txt`.
 To check the types execute
 ```
-pytype rasa
+make types
```

Comment:
what's the advantage from having `make` here instead of the library itself?

Reply:
The idea is that if we want to modify the parameters we pass to pytype some day, we would only need to do it in one place (Makefile). Or even if we want to change the type checker for another one.

Reply:
I'm convinced :-)